app/jobs/scraper/new_york_job.rb

```diff
@@ -0,0 +1,42 @@
+class Scraper::NewYorkJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    url = "https://dol.ny.gov/apprenticeship/apprenticeship-trades"
+    uri = URI.parse(url)
+    response = Net::HTTP.get_response(uri)
+    html = response.body
+    doc = Nokogiri::HTML(html)
+
+    table_and_nav = doc.css("div.wysiwyg--field-webny-wysiwyg-body")
+    table = table_and_nav.css("tbody")[0]
+    table.css("tr").each do |row|
+      next unless row.css("a").present?
+      file_name = row.css("a").first["href"]
+      file_path = ""
+      if /https/.match?(file_name)
+        file_path = file_name.gsub("https://dol.ny.gov/system/files/documents/2022/06", "").gsub(".pdf", "")
```

Comment:
Can you make `https://dol.ny.gov/system/files/documents/2022/06` a constant, since it is used in two places?